web/src/app/main/components/homepage/homepage.component.html

```diff
@@ -76,7 +74,10 @@
           <ng-container matColumnDef="icon">
             <th mat-header-cell *matHeaderCellDef></th>
             <td mat-cell *matCellDef="let element">
-              <mat-icon class="home__users-card__content__table__icon material-icons-outlined green-color">{{ checkTypeOfProject(element) }}</mat-icon>
+              <img *ngIf="element.logoUrl; else imagePlaceholder" src="{{element.logoUrl}}" />
```

Comment:
It is better to have the src like `[src]="element.logoUrl"`

Reply:
okay

Reply:
You did this in some places, I fixed the other places